_notices/rsn0030.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 30 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Docker Image Changes in Release v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Projgess
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"
+notice_created: 2023-04-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-04-20
+---
+
+## Overview
+
+RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be introduced in the RAPIDS 23.06 release.
+
+While many of these changes should be seamless to users, some of them may be breaking. These breaking changes include:

```

Comment:
Should this be a bulleted list?

Reply:
Thats a good point, but we should be very clear in this RSN that there are breaking changes and that users should review this list.